src/Diagnostics.Logger/ETWProvider/DiagnosticsETWProvider.cs

```diff
@@ -219,6 +219,21 @@ public void LogDataProviderOperationSummary(string RequestId, string Source, str
                 EndTime,
                 LatencyInMilliseconds);
         }
+
+        [Event(3003, Level = EventLevel.Warning, Channel = EventChannel.Admin, Message = ETWMessageTemplates.LogKustoTokenRefreshSummary)]
```

Comment:
Should this event just be info level? If it is going to be healthy most of the time.

Reply:
yes. good catch. (copy paste error)